fiftyone/utils/torch.py

```
@@ -2605,6 +2667,244 @@ def _parse_inputs(
+
+        return decoder, iterator
+
+    def _cleanup_current_decoder(self):
+        """Close current decoder if any."""
+        if self._current_decoder is not None:
+            self._current_decoder.close()
+            self._current_decoder = None
+            self._current_iterator = None
+            self._current_video_path = None
+
+    def _extract_sample_data(self, samples):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_extract_sample_data`

Add return type annotation: `None`

(ANN202)